cobalt/browser/application.cc

```diff
@@ -732,6 +732,20 @@ Application::Application(const base::Closure& quit_closure, bool should_preload,
     options.renderer_module_options.enable_fps_overlay = true;
   }
 
+  if (command_line->HasSwitch(browser::switches::kEnableSkiaRasterizer)) {
```

Comment:
I'm okay with this going in as is, btu this commandline checking logic is now in three separate places now.

All of those places already seem to pull in `cobalt/renderer/get_default_rasterizer_for_platform.h` header - maybe move the function there ? I'm fine either way though.

Reply:
Calling this function from /renderer could create a circular dependency with the other call sites (/image_decoder and /memory_settings) ? Since the eventual goal is to deprecate direct-gles entirely, I think we can remove this and clean it up later.